modules/op-prepare-opc-for-results.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="prepare-opc-for-results_{context}"]
+= Preparing the `opc` utility environment for querying {tekton-results}
+
+[role="_abstract"]
+Before you can query {tekton-results}, you must prepare the environment for the `opc` utility.
+
+.Prerequisites
+
+* You installed {tekton-results}.
+* You installed the `opc` utility.
+
+.Procedure
+
+. Set the `RESULTS_API` environment variable to the route to the {tekton-results} API. Run the following command:
++
+[source, terminal]
+----
+$ export RESULTS_API=$(oc get route  tekton-results-api-service  -n openshift-pipelines  --no-headers -o custom-columns=":spec.host"):443
+----
+
+. To authenticate with the {tekton-results} API, create an authentication token and save it to the `~/.config/tkn/results.yaml` file. Run the following command:
++
+[source, terminal]
+----
+$ create token <service-account> > ~/.config/tkn/results.yaml
+----
++
+Replace `<service-account>` with the name of a service account that is authorized to use {pipelines-shortname}.
++
+Alternatively, if you do not create the `~/.config/tkn/results.yaml` file, you can pass the token to each `opc` command using the `--authtoken` option.
```

Comment:
@avinal is this file created automatically or does one have to create it manually? Also how does one provide a token in this file?

Reply:
No, it is not created automatically, you must manually create it in the given path. You have included the command to create a token in the docs but is wrong, here is the right one.

```
oc create token <service-account>
```
then copy and paste the output.

If you have provided a service account in the file, a token is not needed. But if you do, it will be preferred. It can be useful when you want to query multiple results server.

Reply:
@avinal now I am confused. How is "redirect output to the file" differ from "copy and paste the output to the file"?

Or is this about the fact that I forgot `oc`? Fixing this now.

Reply:
Here is the format
```yaml
 address: <address-of-the-result-apiserver>
token: <paste-the-created-token-here>
ssl:
  roots_file_path: <path-to-the-cert-file>
  server_name_override: tekton-results-api-service.tekton-pipelines.svc.cluster.local
service_account:
  namespace: <namespace-of-the-service-account>
  name: <name-of-the-service-account>
```